test/dlc_tests/sanity/test_security_check.py

```diff
@@ -54,6 +66,297 @@ def run_scan(ecr_client, image):
         raise TimeoutError(f"ECR Scan is still in {scan_status} state. Exiting.")
 
 
+def get_new_image_uri(image):
```

Comment:
Please move all the helper functions into a separate file, so that we can keep only tests here. This test file has gotten too long, now, and seems to mostly be just helper code.

I would recommend moving these to either `test_utils/__init__.py` or `test_utils/security.py`.